packages/x-charts-pro/src/ChartZoomSlider/internals/ChartAxisZoomSlider.tsx

```diff
@@ -387,27 +405,44 @@ function ChartAxisZoomSliderSpan({
         width={previewWidth}
         height={previewHeight}
       />
-      {
-        // TODO: In RTL languages, should we start from the right?
-      }
       <ChartAxisZoomSliderHandle
+        ref={setStartHandleEl}
         x={startHandleX}
         y={startHandleY}
         width={previewHandleWidth}
         height={previewHandleHeight}
         orientation={axisDirection === 'x' ? 'horizontal' : 'vertical'}
-        onResize={onResizeStart}
+        onMove={onResizeStart}
```

Comment:
I just noticed the `onResize[ Start | End ]` is confusing because I can be:
- Modify the start/end position (the correct one)
- start/end modifying positions (like `onDraggStart`)

```suggestion
        onMove={onMoveStartHandle}
```